docs/src/consolePlayground/index.js

```diff
@@ -1,10 +1,12 @@
 import * as falso from '../theme/ReactLiveScope/falso.min';
 
 window.onload = function () {
-	const titleStyles = 'color:#8a76d9; font-size: 22px; font-weight:bold;';
-	const copyStyles = 'color:white; font-size: 14px';
-	const exampleStyles = 'color:#8a76d9; font-size: 14px;';
-	console.log('%cFalso\n\n' + '%cYou can use _ to access Falso functions here in the console. \n\n%c' + 'For example: _.randFullName()', titleStyles, copyStyles, exampleStyles)
+	if (typeof window) {
```

Comment:
if (typeof window !== 'undefined')

Reply:
😬 Sorry mate

Reply:
3rd time lucky 🤞

Reply:
You're right, missed this. I'm with Covid these days 😂

Reply:
@theryansmee wrap the entire block and merge, please